### PR TITLE
Component setup

### PR DESCRIPTION
**Pull request by Chriscastanuela**
### Description
1. Makes it so that when a page is clicked the corresponding icon turns blue
2. New `Feed.js` component that is dynamically used for home (feed) and user posts page
3. Flexbox used in `Feed.js`
### How Has This Been Tested?
Ran in DOM

### Screenshots (if appropriate):
flexbox
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
### Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: src/Feed/Feed.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        <section className='feed'>
+            <Post/>
+            <Post/>
+            <Post/>
+            <Post/>
+            <Post/>
+            <Post/>
+            <Post/>
+            <Post/>
+            <Post/>
```
Comment (MichaelEWalker87) [inline src/Feed/Feed.js:16]: is just a temporary thing until we can loop over the data

File: src/Header/Header.js [hunk 2]
```diff
@@ -1,34 +1,30 @@
...
+          <img src={props.icons.post} alt='Add a post' className='bottom-images' id='make-post-icon'/>
+        </Link>
+        <Link to="/my_post" label="my post" className='icon-links'>
+          <img src={props.icons.feed} alt='Your feed' className='bottom-images' id='user-feed-icon'/>
+        </Link>
+        <Link to="/awards" label="my awards" className='icon-links'>
+          <img src={props.icons.awards} alt='Your awards' className='bottom-images' id='user-awards-icon'/>
+        </Link>
+      </section>
+    </section>
```
Comment (MichaelEWalker87) [inline src/Header/Header.js:28]: I am glad that you were able to get this connected without any major issues

File: src/App.js [hunk 3]
```diff
@@ -1,47 +1,76 @@
...
+  constructor() {
+    super()
+    this.state = {
+      page: 'home'
+    }
+  }
+  render() {
+    return (
+      <section>
+        <Route exact path='/' render={ () =>
```
Comment (MichaelEWalker87) [inline src/App.js:29]: I use the exact path Matthew recommended switching to NavLinks I believe

**Review by MichaelEWalker87 — COMMENTED** [review]
I did notice a few things when moving from the mobile to desktop we had it remain a single column molded after how Reddit keeps to a single item. Do we as a group want to continue with the wrapped look if so we can make that happen. 
Thank you for all your hard work I think things are coming along

File: src/App.js [hunk 4]
```diff
@@ -1,47 +1,76 @@
...
+          </section>}
+        />
+        <Route exact path='/awards' render={ () =>
+          <section className="awards">
+            <Header icons={{
+              home: homeWhite,
+              post: postWhite,
+              feed: feedWhite,
+              awards: awardsBlue
+            }}/>
```
Comment (GreyMatteOr) [inline src/App.js:69]: Haha I was wondering where this logic was being stored. Are we just ethically opposed to Navlinks?? XD

File: src/requests.js [hunk 5]
```diff
@@ -1,39 +1,39 @@
...
-})
-.then(result => console.log(result));
+// client.query({
+//   query: gql`
+//     mutator createPost($lat: lat, $long: long, $content: content) {
+//       ring(lat: $lat, long: $long, content: $content)
+//     }
+//   `
+// })
+// .then(result => console.log(result));
```
Comment (GreyMatteOr) [inline src/requests.js:27]: Sorry! Thanks for commenting everything out.

**Review by GreyMatteOr — APPROVED** [review]

**Pull request closed without merging**